### DISCUSSION TITLE
Newbie Q

### DISCUSSION THREAD

**Discussion by robzz33** [witnessmenow/ESP32-Cheap-Yellow-Display#255] [General]
Hi all!

I stumbled on to the CYD by sheer dumb luck, and I'm glad I did! this community seems to be alive and kicking

So... I'm very VERY green to this whole world of CYD, platformIO, etc but very keen on getting familiar with it.
I can navigate VS code just enough to modify 3d printer setting for marlin. that's the extent of my knowledge

I'm trying to make a project, a simple project using the CYD to do the task of a Rotary encoder, so basically digitizing the analogue encoder.

All I'm looking to do is have a splash screen, (possibly a cool animated one)  when the ESP starts up then on to 6 buttons that will mimic the encoder position.

my question is, is this something that would be possible with just the CYD or will I need more GPIOs?

cheers

Robzz

**Comment by Dilosen** [comment]
Look at the examples and then work from there, you don't need any additional GPIO to test unless you are using sensors or controllers or something like that
